ansible/roles/cluster_infra/templates/resources.tf.j2

```diff
@@ -79,6 +79,22 @@ resource "openstack_blockstorage_volume_v3" "state" {
     size = "{{ state_volume_size }}"
 }
 
+{% if cluster_home_manila_share | bool %}
+resource "openstack_sharedfilesystem_share_v2" "home" {
+  name = "{{ cluster_name }}-home"
+  description = "Home for control node"
+  share_proto = "CEPHFS"
+  share_type = {{ '"' + cluster_home_manila_share_type + '"' | default('null') }}
+  size = "{{ home_volume_size }}"
+}
+
+resource "openstack_sharedfilesystem_share_access_v2" "home" {
+  share_id     = openstack_sharedfilesystem_share_v2.home.id
+  access_type  = "cephx"
+  access_to    = "{{ cluster_id }}" # TODO: is this guranteed to be unique across all projects? Presumably?
```

Comment:
cluster_id should work nicely here, I think.

Only one extra idea is to try "cluster_{{ cluster_id }}" maybe? just so its clearer to humans?